presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -131,15 +141,20 @@ public OperatorFactory duplicate()
     private boolean closed;
     private long writtenBytes;
 
-    public TableWriterOperator(OperatorContext operatorContext,
+    public TableWriterOperator(
+            OperatorContext operatorContext,
             ConnectorPageSink pageSink,
-            List<Integer> inputChannels)
+            List<Integer> columnChannels,
+            Operator statisticAggregation,
+            List<Type> types)
     {
         this.operatorContext = requireNonNull(operatorContext, "operatorContext is null");
         this.pageSinkMemoryContext = operatorContext.newLocalSystemMemoryContext();
         this.pageSink = requireNonNull(pageSink, "pageSink is null");
-        this.inputChannels = requireNonNull(inputChannels, "inputChannels is null");
+        this.columnChannels = requireNonNull(columnChannels, "columnChannels is null");
```

Comment:
Renamed because it's input + column stats channels?

Reply:
Yup